### DISCUSSION TITLE
league.box_scores() problem says key winner error

### DISCUSSION THREAD

**Discussion by amirbh1106** [cwendt94/espn-api#298] [ESPN API]
`year = 2022

league = League(league_id=leagueId, year=year, espn_s2=s2, swid=swid1)

team = league.teams[0]

h = league.box_scores()

print(h)
`

its says 
`
   self.winner = data['winner']
KeyError: 'winner
`

**Comment by cwendt94** [comment]
What sport are you seeing this error in?

**Reply by cb4lenor** [reply]
i see the same error in Basketball when retrieving boxscores ..

**Reply by cwendt94** [reply]
could you post the whole error log and what type of scoring league is it? I am not getting this error for my league.

**Reply by cb4lenor** [reply]
happy new year!

its a total season points league ... 

```
Traceback (most recent call last):
  File "/Users/league.py", line 6, in <module>
    print(myleague.box_scores())
  File "/opt/homebrew/lib/python3.9/site-packages/espn_api/basketball/league.py", line 170, in box_scores
    box_data = [BoxScore(matchup, pro_schedule, matchup_total, self.year) for matchup in schedule]
  File "/opt/homebrew/lib/python3.9/site-packages/espn_api/basketball/league.py", line 170, in <listcomp>
    box_data = [BoxScore(matchup, pro_schedule, matchup_total, self.year) for matchup in schedule]
  File "/opt/homebrew/lib/python3.9/site-packages/espn_api/basketball/box_score.py", line 6, in __init__
    self.winner = data['winner']
KeyError: 'winner'

```

**Reply by cwendt94** [reply]
Oh okay, I have not been in a total season points league before! How do matchups/box scores work with total point seasons? I can fix the key error just not sure how all the other fields will be populated in that type of scoring league!

**Reply by cb4lenor** [reply]
Basically there are no real matchups. Points per players unbenched are summarized and games played are counted. I tried to find a way to get points per team per day and on top games played (per day or total) in order to calculate projections and do some nice graphs ... I played around with the ESPN APIs directly, but I guess its not as easy as it sounds, as the total points leagues seemingly were not the original intent ...

**Reply by cwendt94** [reply]
key error should be fixed in package v0.19.1, hopefully the rest of the box_scores fields are there.
